README.md

```diff
@@ -64,6 +64,7 @@ There is also a new sub-action, `actions/upload-artifact/merge`. For more info,
     Due to how Artifacts are created in this new version, it is no longer possible to upload to the same named Artifact multiple times. You must either split the uploads into multiple Artifacts with different names, or only upload once. Otherwise you _will_ encounter an error.
 
 3. Limit of Artifacts for an individual job. Each job in a workflow run now has a limit of 500 artifacts.
+4. With `v4.4` and later, hidden files are excluded by default.
```

Comment:
Why are we introducing this for 4.4+ only?

Reply:
This section of the `README` corresponds to breaking changes. We wrote this when there a ton of breaking changes between v3 and v4. With this new "default" behavior, we are introducing a new breaking change. Perhaps this can be expanded to clarify the behavior. What about something like:

```
With `v4.4` and later, hidden files are excluded by default. All prior versions of upload-artifact included hidden files by default
```

Reply:
What about GHES users where only v3 works (with actions/artifact lib v1)?

Where GHES is going to support v4 btw?

Reply:
We are going to backport a change to v3 as there is no timeline currently for v4 to be in GHES.